<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>secondwindow</class>
 <widget class="QDialog" name="secondwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>308</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="Btn1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>40</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn5">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn6">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>130</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn8">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>220</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn9">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>220</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="DisplayTurn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
